src/ui/components/Timeline/index.tsx

```diff
@@ -88,7 +88,6 @@ class Timeline extends Component<PropsFromRedux> {
     const mouseTime = this.getMouseTime(e);
 
     if (hoverTime != mouseTime) {
-      paintGraphicsAtTime(mouseTime);
```

Comment:
moved an incidental graphics call to a timeline action so the logic would be in one place